diff --git a/dist/index.js b/dist/index.js
index c9c5838142fb1b7026b65d6074816cea8fe9eb2e..8111917bb807e67089817b595d4c6f962e8971d8 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -32,7 +32,7 @@ __export(src_exports, {
 module.exports = __toCommonJS(src_exports);
 
 // src/render.ts
-var ReactDomServer = __toESM(require("react-dom/server"));
+var ReactDomServer = __toESM(require("react-dom/server.browser"));
 var import_html_to_text = require("html-to-text");
 var import_pretty = __toESM(require("pretty"));
 var render = (component, options) => {
@@ -59,7 +59,7 @@ var renderAsPlainText = (component, _options) => {
 // src/renderAsync.ts
 var import_html_to_text2 = require("html-to-text");
 var import_pretty2 = __toESM(require("pretty"));
-var import_server = require("react-dom/server");
+var import_server = require("react-dom/server.browser");
 async function renderToString(children) {
   const stream = await (0, import_server.renderToReadableStream)(children);
   const html = await readableStreamToString(
diff --git a/dist/index.mjs b/dist/index.mjs
index 27cc390481a1847fba0008ffad54c45a23a24f27..358db5cba96bd9286e0f60635550efdb0dae5c64 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,5 +1,5 @@
 // src/render.ts
-import * as ReactDomServer from "react-dom/server";
+import * as ReactDomServer from "react-dom/server.browser";
 import { convert } from "html-to-text";
 import pretty from "pretty";
 var render = (component, options) => {
@@ -26,7 +26,7 @@ var renderAsPlainText = (component, _options) => {
 // src/renderAsync.ts
 import { convert as convert2 } from "html-to-text";
 import pretty2 from "pretty";
-import { renderToReadableStream, renderToStaticMarkup as renderToStaticMarkup2 } from "react-dom/server";
+import { renderToReadableStream, renderToStaticMarkup as renderToStaticMarkup2 } from "react-dom/server.browser";
 async function renderToString(children) {
   const stream = await renderToReadableStream(children);
   const html = await readableStreamToString(
